Sierra/challenge_14/stations_challenge.py

```diff
@@ -0,0 +1,111 @@
+'''
+
+
+from binhex import openrsrc
+
+
+print('Question 1: Making the SubwayStation Class')
+'''
+Using the Station class below as the parent, make a child class called SubwayStation
+
+SubwayStation should:
+-have an additional attribute called 'lines' that is user-defined as a list during initialization. 
+    this will indicate which subway lines stop at the station (for example ['A', 'C'])
+-override the show_info() method from Station to display which subway lines stop there, in addition to the station_name and location
+'''
+class Station:
+    def __init__(self, station_name, location):
+        self.station_name = station_name
+        self.location = location
+    
+    def show_info(self):
+        print(f'{self.station_name} station is located at {self.location}')
+
+
+class SubwayStation(Station):
+    def __init__(self, lines, station_name, location):
+        super().__init__(station_name, location)
+        self.lines = lines
+
+    def show_info(self):
+        print(f'{self.lines} are the lines that stop at {self.station_name} station. {self.station_name} station is located at {self.location}.')
+
+
+print('Question 2: Make an example subway station')
+'''
+Using your SubwayStation class, instantiate a subway station with the info below. 
+Then run the show_info() method to make sure you get the station_name, location, and lines printed out
+
+station_name: '14th street'
+location: '14th street and 7th avenue'
+lines: ['1', '2', '3', 'L']
+'''
+subway_station1 = SubwayStation(station_name='14th street', location='14th street and 7th avenue', lines=['1', '2', '3', 'L'])
+
+subway_station1.show_info()
+
+print('Question 3: Making the BusStation Class')
+
+'''
+Using the Station class below as the parent, make a child class called BusStation
+
+BusStation should:
+-have an additional attribute called 'routes' that is user-defined as a list during initialization. 
+    this will indicate where buses can go from this station. For example, ['DC', 'Philly', 'Pittsburgh']
+-have an additional attribute called 'open' that is always initalized as True (a boolean variable)
+-have additional methods called open_station() and close_station() to open and close the station
+-override the show_info() method from Station to display the bus routes and if the station is open, in addition to the station name and location
+'''
+class BusStation(Station):
+    def __init__(self, routes, station_name, location):
+        super().__init__(station_name, location)
+        self.routes = routes
+        self.open = True
+    
+    def open_station(self):
+        self.open == True
+        self.open = 'open'
+
+    def close_station(self):
+        self.open == False
+        self.open = 'closed'
+
+    def show_info(self):
+        self.open_station()
+        print(f'The bus routes are {self.routes} and the station is {self.open}. {self.station_name} is located at {self.location}.')
+
```

Comment:
What the instruction meant to do is for you to use an if else statement to check whether the station is open or not. If self.open == True, then you print out the statement, otherwise, the boolean declaration lost its purpose.